pkg/apis/provisioning/v1alpha5/requirements.go

```diff
@@ -25,6 +25,7 @@ var (
 	ArchitectureAmd64    = "amd64"
 	ArchitectureArm64    = "arm64"
 	OperatingSystemLinux = "linux"
+	BottleRocket         = "bottlerocket"
```

Comment:
We try to keep AWS specific concepts isolated to the cloudprovider/aws package, check out the APIs.

Reply:
nit: Should this be `OperatingSystemBottleRocket` to match the linux var

Reply:
moved this to cloudprovider/aws/apis/v1alpha1/register.go, renamed to OperatingSystemBottleRocket.